src/components/landingPage/LandingPage.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Button from '@material-ui/core/Button';
+import {Link} from 'react-router-dom';
+import {useStyles} from '../LayoutStyles';
+import mockAuth from '../Authenticator';
+
+export default function LandingPage() {
+  const classes = useStyles();
+
+  /* This part will be fetched from server */
+  let landingPageContent = [
+    {to: '/self-service', label: 'Self service page'},
+    {to: '/restricted-area', label: 'Restricted area'}];
+  if (mockAuth.isAdmin()) {
+    landingPageContent.push({to: '/admin', label: 'Admin panel'});
+  }
+  /* End of part fetched from the server */
```

Comment:
nit: End of part *to be* fetched from the server

Reply:
Done.